app/src/main/kotlin/com/wire/android/ui/common/topappbar/search/SearchTopBar.kt

```diff
@@ -159,7 +159,7 @@ private fun animateHorizontalAlignmentAsState(
 ): State<BiasAlignment.Horizontal> {
     val biased = targetAlignment as BiasAlignment
     val bias by animateFloatAsState(biased.horizontalBias, label = "AnimateHorizontalAlignment")
-    return derivedStateOf { BiasAlignment.Horizontal(bias) }
+    return remember { derivedStateOf { BiasAlignment.Horizontal(bias) } }
```

Comment:
maybe remove the @SuppressLint("UnrememberedMutableState")

Reply:
right, I changed it but missed the annotation, now it's removed